### PR TITLE
release: highlights of v0.0.8

### PR DESCRIPTION
**Pull request by aslakknutsen**
/skip-e2e

Signed-off-by: Aslak Knutsen <aslak@4fs.no>

### CODE REVIEW

File: docs/modules/ROOT/pages/release_notes/v0.0.8.adoc [hunk 1]
```diff
@@ -0,0 +1,39 @@
+== Highlights of v0.0.8 release
+
+=== Watch dependent resources for change
+
+The operator will now watch all the dependent resources (DestinationRule, Gateway, VirtualService, Deployment)
```
Comment (bartoszmajsak) [inline docs/modules/ROOT/pages/release_notes/v0.0.8.adoc:5]:
```suggestion
The operator will now watch all the dependent resources (`DestinationRule`, `Gateway`, `VirtualService`, `Deployment`)
```

**Comment by alien-ike** [comment]
### ℹ️ Available commands

 * `/changelog vX.Y.Z` will append changelog based on the closed PRs since last release to provided release notes.

 * `/release vX.Y.Z` will add release-related commits to this PR on your behalf. 

 * `/shipit` will rebase everything on master branch and trigger actual release process

File: docs/modules/ROOT/pages/release_notes/v0.0.8.adoc [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+and reconcile the Session on any change.
```
Comment (bartoszmajsak) [inline docs/modules/ROOT/pages/release_notes/v0.0.8.adoc:6]:
```suggestion
and reconcile the `Session` on any change.
```

File: docs/modules/ROOT/pages/release_notes/v0.0.8.adoc [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+
+This means that even if a 3. party tool reconfigures the current state, overwrites it, or pushes new releases, the operator will ensure that the special route is still operational.
```
Comment (bartoszmajsak) [inline docs/modules/ROOT/pages/release_notes/v0.0.8.adoc:8]:
```suggestion
This means that even if a third-party tool reconfigures the current state, overwrites it, or pushes new releases, the operator will ensure that the special route is still operational.
```

**Review by bartoszmajsak — CHANGES_REQUESTED** [review]
Minor wordings suggestion (bulk update with `/skip-e2e`)

**Comment by aslakknutsen** [comment]
@bartoszmajsak merged, rebased and squashed..

**Review by bartoszmajsak — APPROVED** [review]

**Comment by alien-ike** [comment]
🎇 🎉 🎆 New version has been shipped! 🚀🚀🚀

**Pull request merged**